########################################
# Messages
########################################
uint8 seq_num
bool ready